src/sdb.c

```diff
@@ -717,9 +723,9 @@ SDB_API int sdb_expire_set(Sdb* s, const char *key, ut64 expire, ut32 cas) {
 
 SDB_API ut64 sdb_expire_get(Sdb* s, const char *key, ut32 *cas) {
 	SdbKv *kv;
-	ut32 hash = sdb_hash (key);
-	kv = (SdbKv*)ht_lookup (s->ht, hash);
-	if (kv && *kv->value) {
+	bool found;
```

Comment:
this is because if you want to store intptr_t as the values you may want to be able to return NULL. this allows you to store NULL in the ht as a value.